packages/gestalt-datepicker/src/DatePickerTextField.js

```diff
@@ -0,0 +1,115 @@
+// @flow strict
+import * as React from 'react';
+import { Box, Icon, Label, TextField } from 'gestalt';
+import PropTypes from 'prop-types';
+
+// InjectedProps are props that Datepicker adds on to DatePickerTextField.
+// Datepicker takes this props and then funnels them to DatePickerTextField.
+type InjectedProps = {|
+  disabled?: boolean,
+  id: string,
+  onBlur?: (event: SyntheticFocusEvent<HTMLInputElement>) => void,
+  onChange?: (event: SyntheticInputEvent<HTMLInputElement>) => void,
+  onClick?: () => void,
+  onFocus?: (event: SyntheticFocusEvent<HTMLInputElement>) => void,
+  onKeyDown?: (event: SyntheticKeyboardEvent<HTMLInputElement>) => void,
+  placeholder?: string,
+  value?: string,
+|};
+
+type OwnProps = {|
+  accessibilityLabelIcon: string,
+  isRTL?: boolean,
+  id: string,
+  ...InjectedProps,
+|};
+
+type Props = {|
+  ...OwnProps,
+  forwardedRef?: React.ElementRef<*>,
+|};
+
+function DatePickerTextField(props: Props) {
+  const {
+    accessibilityLabelIcon,
+    disabled,
+    forwardedRef,
+    id,
+    isRTL = false,
+    onChange,
+    onClick,
+    onBlur,
+    onFocus,
+    onKeyDown,
+    placeholder,
+    value,
+  } = props;
+
+  return (
+    <div role="none" onClick={onClick}>
+      <Label htmlFor={id}>
+        <Box
+          alignItems="center"
+          column={12}
+          flex="grow"
+          display="flex"
+          position="relative"
+        >
+          <Box column={12} flex="grow">
+            <TextField
+              autoComplete="off"
+              disabled={disabled}
+              id={id}
+              onBlur={data => onBlur && onBlur(data.event)}
+              onFocus={data => onFocus && onFocus(data.event)}
+              onChange={data => onChange && onChange(data.event)}
+              onKeyDown={data => onKeyDown && onKeyDown(data.event)}
+              placeholder={placeholder}
+              ref={input => forwardedRef && forwardedRef(input || null)}
+              size="lg"
+              value={value}
+            />
+          </Box>
+          <Box position="absolute" right={!isRTL} left={isRTL}>
```

Comment:
@AlbertCarreras let's remove the isRTL prop

Reply:
fixed